p2p/discovery/discovery.go

```diff
@@ -0,0 +1,116 @@
+// Package discovery implements a Distributed Hash Table based on Kademlia protocol.
+package discovery
+
+import (
+	"context"
+	"errors"
+	"github.com/spacemeshos/go-spacemesh/p2p/config"
+	"github.com/spacemeshos/go-spacemesh/p2p/node"
+	"github.com/spacemeshos/go-spacemesh/p2p/p2pcrypto"
+	"github.com/spacemeshos/go-spacemesh/p2p/server"
+)
+
+// DHT is an interface to a general distributed hash table.
+type Interface interface {
```

Comment:
maybe use api or something , because interface is a golang keyword i think it will be clearer